tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1391,6 +1391,44 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x08 - IV Update test mode
+		Controller Index:	<controller id>
+		Command parameters:	Enable (1 octet)
+
+		This command is used to enable/disable the IV Update test mode.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x09 - IV Update toggle state
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	State (1 octet)
+
+		Valid State response values:
+					0x00 = Normal
+					0x01 = IV Update in Progress
+
+		This command is used to make a IV Update state transition
+		while in IV Update test mode, ignoring the 96 hour limit.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x0a - Network Send
+		Controller Index:	<controller id>
+		Command parameters:	Valid parameters (1 octet)
+					TTL (1 octet)
+					SRC (2 octets)
+					DST (2 octets)
```

Comment:
Isn't there the actual payload parameter missing here?

Reply:
I'll explain myself ;) I thought about adding this, but I saw that PTS is not for any particular data to be sent, so I just skipped this and I wanted to add payload in handler of this command.
But on the other hand this command would be more versatile when payload will be here.